pages.zh/common/tar.md

```diff
@@ -0,0 +1,37 @@
+# tar
+
+> 归档实用程序。
+> 通常与压缩方法结合使用，例如 gzip 或 bzip2。
```

Comment:
I know this is somewhat strange in Chinese, but it's the current practice in tldr to use half-width punctuation when the tail of a sentence is in English.
```suggestion
> 通常与压缩方法结合使用，例如 gzip 或 bzip2.
```